src/GraphQL.Tests/Introspection/IntrospectionResult.cs

```diff
@@ -1,7 +1,7 @@
+        public static readonly string Data = "{\r\n  \"__schema\": null\r\n}";
```

Comment:
Hm, this change is a little troubling.  Didn't notice this one on the failed tests.  Will look at this closer.

Reply:
You can find failed tests [here](https://ci.appveyor.com/project/graphql-dotnet-ci/graphql-dotnet/build/2.0.0.863-dxxhbeqj/tests). Failed test name is `GraphQL.Tests.Introspection.SchemaIntrospectionTests.validate_core_schema`.
As I see, `queryType` property is NonNullable, but its value is `null`. So, its value should be bubbled to nearest nullable parent - `__shema`.